.travis/osx..script.sh

```diff
@@ -5,4 +5,4 @@ if [ $QT5 ]; then
         export CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
 fi
 
-cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWANT_QT5=$QT5 -DUSE_WERROR=OFF ..
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DWANT_QT5=$QT5 -DUSE_WERROR=ON ..
```

Comment:
`USE_WERROR` is not strictly necessary because you can use `CFLAGS=-Wno-error` and `CXXFLAGS=-Wno-error`. `USE_WERROR` should be dropped if Clang warnings go away.